config/urls.py

```diff
@@ -2,15 +2,19 @@
 from rest_framework import routers
 from django.contrib import admin
 from django.urls import path
+from apps.goals.views import GoalViewSet
+from apps.user.v1.views import UserViewSet
+
 
 router = routers.DefaultRouter()
+router.register(r'user', UserViewSet)
```

Comment:
r'user' what this `r` is used for?

Reply:
It means raw string: 

Eg 
```py 
s = 'This is a normal string with an escape sequence: \nNew line'
print(s)
# Output: This is a normal string with an escape sequence: 
# New line

raw_s = r'This is a raw string with no escape sequence: \nNew line'
print(raw_s)
# Output: This is a raw string with no escape sequence: \nNew line
```

It is the recommended way in the [docs](https://www.django-rest-framework.org/api-guide/routers/)